### ISSUE TITLE
lab_10_5 dropout

### ISSUE THREAD

**Issue by cjr0106** [hunkim/DeepLearningZeroToAll#210]
why the dropout keep 0.7? where could get the default 
![image](https://user-images.githubusercontent.com/32252431/42302345-0274928a-804d-11e8-8140-4a53777f89a9.png)

**Comment by kkweon** [comment]
The `keep_prob` is a hyperparameter, meaning you have to find the best value and store into that variable.

`tf.placeholder` is simply a variable. 
There is no such thing as 0.7 being the default parameter.
Later in the same file, you can see we're setting 0.7 to that placeholder in fact.

```python
    for i in range(total_batch):
        batch_xs, batch_ys = mnist.train.next_batch(batch_size)

        # Set keep_prob to 0.7
        feed_dict = {X: batch_xs, Y: batch_ys, keep_prob: 0.7}

        c, _ = sess.run([cost, optimizer], feed_dict=feed_dict)
        avg_cost += c / total_batch
 ```

**Comment by cjr0106** [comment]
Thanks so much ,i got it~

2018-07-06 01:42:58"Mo Kweon" <notifications@github.com>写道：

The keep_prob is a hyperparameter, meaning you have to find the best value and store into that variable.

tf.placeholder is simply a variable.
There is no such thing as 0.7 being the default parameter.
Later in the same file, you can see we're setting 0.7 to that placeholder in fact.

for i inrange(total_batch):
        batch_xs, batch_ys = mnist.train.next_batch(batch_size)

        # Set keep_prob to 0.7
        feed_dict = {X: batch_xs, Y: batch_ys, keep_prob: 0.7}

        c, _ = sess.run([cost, optimizer], feed_dict=feed_dict)
        avg_cost += c / total_batch
